src/main/webapp/creatives/maps/interaction.js

```diff
@@ -0,0 +1,45 @@
+
+import {generateSessionId} from '../maps/utils.js';
+
+const URL = {
+  INITIALIZE: "initialize",
+  FIND_NEAREST_LOCATION: "find-location",
+  GRANT_LOCATION_DATA: "grant-location",
+  MAP_INTERACT: "map-interact",
+  SKIP_TO_CONTENT: "skip-to-content",
+  RETURN_TO_AD: "return-to-ad"
+}
+
+class User {
+  constructor() {
+    this.correlator = generateSessionId();
+  }
+
+  sendRequest_(url) {
+    fetch(`/${url}?correlator="${this.correlator}"`).then(() => console.log("success"));
+  }
+
+  initialize() {
```

Comment:
I would comment these functions.  Or maybe rename them.

logUserInitialized
logUserClickFindNearestLocation

or potentially if this whole class is renamed to logger or statsLogger, your use would make sense too when you call it.

For Example:
logger.userInitialized();

or
statsCollection.onUserInitialize();

way more understandable what exactly is happening compared to:
user.initialize()

Reply:
Renamed class to Logger and also renamed the methods!